google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/JsonStreamWriter.java

```diff
@@ -350,6 +354,16 @@ public Builder setTraceId(String traceId) {
       return this;
     }
 
+    /**
+     * Ignores fields unknown to BigQuery in Json input.
+     *
+     * @return Builder
+     */
+    public Builder ignoreUnknownFields() {
```

Comment:
I think it's more conventional and explicit(true vs. false) if we do `setIgnoreUnknownFields(boolean ignoreUnknwonFields) {...}` ([an equivalent example](https://github.com/googleapis/java-bigquery/blob/main/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/QueryJobConfiguration.java#L497-L499) in google-cloug-bigquery client).

Reply:
Done.